packages/core/src/extensions/Hyperlinks/menus/helpers/PanelTextInputStyles.ts

```diff
@@ -1,8 +1,10 @@
-import { N400, N800 } from "@atlaskit/theme/colors";
 import styled from "styled-components";
 
 // code taken from https://bitbucket.org/atlassian/design-system-mirror/src/master/editor/editor-core/src/ui/PanelTextInput/styles.ts
 
+export const N400 = "#505F79";
```

Comment:
(about line 1)

Let's remove the dependency on styled-components, as mantine uses emotion, and there's no reason to use both afaik

Reply:
Agreed, I'll wrap the `PanelTextInput` component in a `MantineProvider`.